packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -386,55 +402,53 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
             onMouseMove={onMouseMove}
             onMouseUp={onMouseUp}
             onClick={onClick}
-            {...selectedBoxStyle}
+            selectedBoxStyle={selectedBoxStyle}
           />
         )}
         {/* handles */}
         {start &&
           end &&
-          Object.keys(handles)
-            // @ts-ignore
+          (Object.keys(handles) as ResizeTriggerAreas[])
             .filter(handleKey => resizeTriggerAreaSet.has(handleKey))
             .map(handleKey => {
               const handle = handles[handleKey];
 
               return (
-                <BrushHandle
-                  key={`handle-${handleKey}`}
-                  type={handleKey}
-                  handle={handle}
-                  stageWidth={stageWidth}
-                  stageHeight={stageHeight}
-                  updateBrush={this.update}
-                  brush={this.state}
-                  onBrushEnd={onBrushEnd}
-                />
+                handle && (
+                  <BrushHandle
+                    key={`handle-${handleKey}`}
+                    type={handleKey}
+                    handle={handle}
+                    stageWidth={stageWidth}
+                    stageHeight={stageHeight}
+                    updateBrush={this.updateBrush}
+                    brush={this.state}
+                    onBrushEnd={onBrushEnd}
+                  />
+                )
               );
             })}
         {/* corners */}
         {start &&
           end &&
-          Object.keys(corners)
-            // @ts-ignore
+          (Object.keys(corners) as ResizeTriggerAreas[])
             .filter(cornerKey => resizeTriggerAreaSet.has(cornerKey))
             .map(cornerKey => {
               const corner = corners[cornerKey];
 
               return (
-                <BrushCorner
-                  key={`corner-${cornerKey}`}
-                  type={cornerKey}
-                  brush={this.state}
-                  updateBrush={this.update}
-                  stageWidth={stageWidth}
-                  stageHeight={stageHeight}
-                  x={corner.x}
-                  y={corner.y}
-                  width={handleSize}
-                  height={handleSize}
-                  onBrushEnd={onBrushEnd}
-                  fill="transparent"
-                />
+                corner && (
+                  <BrushCorner
+                    key={`corner-${cornerKey}`}
+                    type={cornerKey}
+                    brush={this.state}
+                    updateBrush={this.updateBrush}
+                    stageWidth={stageWidth}
+                    stageHeight={stageHeight}
+                    corner={corner}
```

Comment:
I think corners previously did not work. I moved the coords into an object like `handle` does.